app/controllers/events_controller.rb

```diff
@@ -7,4 +7,20 @@ def index
       render json: collection_events, status: 200
    end
   end
```

Comment:
The 2 ends are alright here?

Reply:
End on row 9 is connected to the index definition.
End on row 8 is connected to the if statement.

You can see this by clicking on the three dots, then view file to see the entire file.